Gruntfile.js

```diff
@@ -208,40 +209,78 @@ module.exports = function(grunt) {
                     config,
                 },
             },
-            copy: {
-                [targetName]: {
-                    files: [
-                        {
-                            cwd: debug ? path.resolve(extensionPath, 'devBundle') : path.resolve(extensionPath, 'prodBundle'),
-                            src: ['*.js', '*.js.map'],
-                            dest: path.resolve(dropExtensionPath, 'bundle'),
-                            expand: true,
-                        },
-                        {
-                            cwd: extensionPath,
-                            src: ['**/*.png', '**/*.css', '**/*.woff'],
-                            dest: dropExtensionPath,
-                            expand: true,
-                        },
-                        {
-                            cwd: 'deploy',
-                            src: ['Gruntfile.js', 'package.json'],
-                            dest: dropPath,
-                            expand: true,
-                        },
-                        {
-                            cwd: extensionPath,
-                            src: ['**/*.html'],
-                            dest: dropExtensionPath,
-                            expand: true,
-                        },
-                    ],
-                },
-            },
             clean: {
                 [targetName]: dropPath,
             },
         });
+
+        if (targetName === 'electron') {
```

Comment:
The two branches of this if look pretty redundant; perhaps it would be better to replace the "debug" config option with a "bundleFolder" or something and use the original version with 		`config.options.bundleFolder` replacing the ternary that was originally on line 215?

(I think this would also simplify the drop task registration further down slightly)

Reply:
The result of applying this idea is so much better. Lest code changes, less bifurcations. Thanks.